clientd3d/d3drender_world.c

```diff
@@ -1246,7 +1254,7 @@ void D3DRenderPacketWallMaskAdd(WallData* pWall, d3d_render_pool_new* pPool, uns
 /*
 * Add a floor mask to the render pool
 */
-void D3DRenderFloorMaskAdd(BSPnode* pNode, d3d_render_pool_new* pPool, Bool bDynamic)
+void D3DRenderFloorMaskAdd(BSPnode* pNode, d3d_render_pool_new* pPool, LPDIRECT3DTEXTURE9 noLookThrough, bool bDynamic)
```

Comment:
Maybe a better name?  Unexpected that "noLookThrough" is a texture, I thought it would be a bool.